src/ray/protobuf/core_worker.proto

```diff
@@ -424,6 +424,16 @@ message ReportGeneratorItemReturnsReply {
   int64 total_num_object_consumed = 1;
 }
 
+message CreateMutableObjectRequest {
+  bytes object_id = 1;
+  int64 num_readers = 2;
+  bytes reader_ref = 3;
```

Comment:
```suggestion
  bytes reader_object_id = 3;
```

(ref usually refers to an ObjectRef or ObjectReference, so "object_id" here is more precise).

Reply:
Please also update the names in node_manager.proto to be consistent.